integration/images/volume-copy-up/Makefile

```diff
@@ -48,7 +48,7 @@ ifneq ($(REMOTE_DOCKER_URL),)
 ALL_OS += windows
 ALL_OSVERSIONS.windows := 1809 20H2 ltsc2022
 ALL_OS_ARCH.windows = $(foreach osversion, ${ALL_OSVERSIONS.windows}, windows-amd64-${osversion})
-BASE.windows := mcr.microsoft.com/windows/nanoserver
```

Comment:
we don't need to update the base image here. just replace the upstream existing images.

Reply:
ok, thanks for the tip.